Competition2018/src/main/java/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -203,8 +203,8 @@ public void setupWristCommands(OperatorInterface oi, WristCalibrateCommand calib
         oi.operatorGamepad.getifAvailable(9).whenPressed(calibrate);
         loseCalibration.includeOnSmartDashboard();
 
-        low.setGoalAngle(0);
-        medium.setGoalAngle(45);
+        low.setGoalAngle(10);
+        medium.setGoalAngle(60);
```

Comment:
After testing, it looks like 10 degrees ends up being slightly nicer than 0 degrees. This may change if we want to rest it on the bumper more explicitly.